src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1093,6 +1109,53 @@ private void EmitSearchSelectionTelemetry(PackageItemViewModel selectedPackage)
             }
         }
 
+        private void IncrementInstalledPackageSelectionCount()
+        {
+            PackageItemViewModel selectedItem = _packageList.SelectedItem;
+            if (selectedItem == null || ActiveFilter != ItemFilter.Installed)
+            {
+                return;
+            }
+
+            if (selectedItem.PackageLevel == PackageLevel.TopLevel)
+            {
+                _installedTabTelemetryData.TopLevelPackageSelectedCount++;
+            }
+            else if (selectedItem.PackageLevel == PackageLevel.Transitive)
+            {
+                _installedTabTelemetryData.TransitivePackageSelectedCount++;
+            }
+        }
+
+        private void PackageList_GroupExpansionChanged(object sender, RoutedEventArgs e)
+        {
+            if (ActiveFilter == ItemFilter.Installed && sender is Expander expander && expander.Tag is PackageLevel pkgLevel)
```

Comment:
If tag is a string, does `is` operator makes a cast?

Reply:
What's being stored in the Tag is the PackageLevel enum so it's not a string. Let me know if that doesn't answer the question! :-)

Reply:
That's ok for me.